src/index.html

```diff
@@ -7,6 +7,366 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="general-container">
+      <header class="header">
+        <a href="#" class="logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">Apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#smartphones" class="nav__link">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#laptops"
+                class="nav__link"
+                data-qa="nav-hover"
+               >
+              Laptops & computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#gadgets" class="nav__link">Gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#tablets" class="nav__link">Tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#pfoto" class="nav__link">Photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#video" class="nav__link">Video</a>
+            </li>
+
+          </ul>
+        </nav>
+      </header>
+
+      <div class="card-container">
+
+        <div class="row-card-first">
+          <div class="card" data-qa="card">
+            <div class="picture-box">
+              <div class="picture-bem1">
+                <img
+                  class="picture-bem1__elem"
```

Comment:
```suggestion
          <div class="card" data-qa="card">
            <div class="card__picture-box">
              <img
                class="card__picture-img"
```
rename all classes according to BEM